Database/Corrections/QuestieQuestFixes.lua

```diff
@@ -230,6 +236,72 @@ function QuestieQuestFixes:Load()
             [QuestieDB.questKeys.sourceItemId] = 18746,
             [QuestieDB.questKeys.zoneOrSort] = 1637,
         },
+        [8289] = { -- #1435
+            [QuestieDB.questKeys.name] = "Talismans of Merit",
+            [QuestieDB.questKeys.startedBy] = {{14733},nil,nil},    --Quest starts and ends with Sentinel Farsong
+            [QuestieDB.questKeys.finishedBy] = {{14733},nil,},
+            [QuestieDB.questKeys.requiredLevel] = 10,               -- All info repeats, but quest has different levels for level of player
+            [QuestieDB.questKeys.questLevel] = 15,
+            [QuestieDB.questKeys.requiredRaces] = 0,                -- All races can partake in quest
```

Comment:
Actually the quests are just for Alliance

```suggestion
            [QuestieDB.questKeys.requiredRaces] = 77,                -- All races can partake in quest
```